.github/workflows/release_workflow.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.9
```

Comment:
Updated to Python 3.9 (it will pick up the latest patch version, which is 3.9.13).